### PR TITLE
chore: 👷workflow for PR title validation

### PR DESCRIPTION
**Pull request by ShwetaChauhan18**
- workflow for PR title validation

### CODE REVIEW

File: .github/workflows/title-validation.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: 'PR Title is Conventional '
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```
Comment (ujas-m-simformsolutions) [inline .github/workflows/title-validation.yml:8]: Also add this type `reopened`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 - Fixed [#253](https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/issues/253) - Add TextAlign attribute for title and description
 - `pull_request_template.md.` file updated with proper document
 - `CONTRIBUTING.md` file updated with proper document
+- PR title validation workflow added
```
Comment (ujas-m-simformsolutions) [inline CHANGELOG.md:8]: Add `added` in front.

**Review by ujas-m-simformsolutions — CHANGES_REQUESTED** [review]

**Review by ujas-m-simformsolutions — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## [2.0.0+2] - December 30, 2022 (Unreleased)
```
Comment (vatsaltanna-simformsolutions) [inline CHANGELOG.md:1]: please change the version to 2.0.1

**Pull request merged**